connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/admin/LoggingResource.java

```diff
@@ -0,0 +1,207 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.connect.runtime.rest.admin;
```

Comment:
Did you consider putting this class in the existing `org.apache.kafka.connect.runtime.rest.resources` package with the other REST resources? I know we haven't had to qualify the different type of resources, but reusing that existing package would allow us to avoid creating a package with a single class, and it would keep all REST resource classes (no matter the category or type) in the same place. WDYT?